AllASPFuncs.py

```diff
@@ -1157,6 +1154,24 @@ def contourGrid(image, numlevels = 5, subsize = 4):
     return y_totalgrid, x_totalgrid
 
 
+def calc_mags_and_err(flux, sigma_flux, band):
+    exptime = {'F184': 901.175,
+            'J129': 302.275,
+            'H158': 302.275,
+            'K213': 901.175,
+            'R062': 161.025,
+            'Y106': 302.275,
+            'Z087': 101.7}
+
+    area_eff = roman.collecting_area
+    galsim_zp = roman.getBandpasses()[band].zeropoint
+    mags = -2.5*np.log10(flux) + 2.5*np.log10(exptime[band]*area_eff) \
+        + galsim_zp
+
+    magerr = 2.5 * sigma_flux / (flux * np.log(10))
```

Comment:
I would have written this as
```
magerr = (2.5 / np.log(10)) * sigma_flux / flux
```
I find these easier to read.  The first term captures the coefficient different between 2.5 log_10 and ln, while the second is really the measurement.